addons/overpressure/CfgEventHandlers.hpp

```diff
@@ -14,7 +14,7 @@ class Extended_PostInit_EventHandlers {
 class Extended_FiredBIS_EventHandlers {
     class CAManBase {
         class ADDON {
-            firedBIS = QUOTE(if (local (_this select 0)) then {_this call FUNC(firedEHBB);};);
+            clientFiredBIS = QUOTE(_this call FUNC(firedEHBB););
```

Comment:
? not sure but if i remeber right is clentFiredBIS only disalbed on server and not local